pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -2161,6 +2161,62 @@ func (a *ApplicationBackupController) cleanupResources(
 		log.ApplicationBackupLog(backup).Errorf("%v", errMsg)
 		return err
 	}
+	// Check if the backup is of the type virtual machine.
```

Comment:
We dont need this comment as the if statement is self explanatory for this comment.  
Suggestion: Cleanup AutoRuleCrs created by stork for virtualMahcine Backup.

Reply:
Removed unnecessary comments